main.py

```diff
@@ -371,3 +372,23 @@ def enable_lldp(self):
         msg_error = "Some interfaces couldn't be found and activated: "
         return jsonify({msg_error:
                         error_list}), 400
+
+    @rest('v1/polling_time', methods=['GET'])
+    def get_time(self):
+        """Get LLDP polling time."""
+        return jsonify({"Polling time in seconds": self.polling_time}), 200
+
+    @rest('v1/polling_time', methods=['POST'])
+    def set_time(self):
+        """Set LLDP polling time."""
+        # pylint: disable=attribute-defined-outside-init
+        try:
+            payload = request.get_json()
+            self.polling_time = int(payload['polling_time'])
+            self.execute_as_loop(self.polling_time)
+            log.info("Polling time has been updated,"
+                     "but this change will not be saved permanently.")
```

Comment:
Please add the new polling time to the log.

Reply:
I added the `polling_time` to log.